client/src/containers/ChooseCampaign.js

```diff
@@ -1,171 +1,193 @@
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
-import { withRouter } from 'react-router';
-import { selectAddress } from '../redux/actions/initialSearch';
-import fetchCampaignById from '../redux/actions/activeCampaign';
-import AutoSuggestInput from '../components/AutoSuggestInput';
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { withRouter } from "react-router";
+import { selectAddress } from "../redux/actions/initialSearch";
+import fetchCampaignById from "../redux/actions/activeCampaign";
+import AutoSuggestInput from "../components/AutoSuggestInput";
+import FullScreenLoader from "../components/FullScreenLoader";
+import cx from "classnames";
 
 class ChooseCampaign extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      selectedOption: null
-    };
-    this.handleOptionChange = this.handleOptionChange.bind(this);
-    this.handleOptionChange = this.handleOptionChange.bind(this);
-    this.handleFormSubmit = this.handleFormSubmit.bind(this);
-  }
+	constructor(props) {
```

Comment:
Spaces -> tabs = less than ideal

Reply:
Just switched in in my workspace settings for this project, so I imagine it'll get corrected from here on out as we go. But you'll never convert me on an ideological level!